recover/recover.go

```diff
@@ -118,6 +116,9 @@ func (r *Recover) StartPost(w http.ResponseWriter, req *http.Request) error {
 		return err
 	}
 
+	ruWithSecondaries, hasSecondaryEmails := authboss.CanBeRecoverableUserWithSecondaryEmails(user)
+
+	ru := authboss.MustBeRecoverable(user)
```

Comment:
Moving this line isn't desirable because you want to prevent firing the events and generating recovery creds.